prime-router/docs/playbooks/dependabot-updates.md

```diff
@@ -22,6 +25,24 @@ When in doubt ask!
    3. Leave a brief comment (i.e. "tested locally"), select Approve and click Submit review
    4. At the bottom of the Conversation tab, click on Update branch, then Enable auto-merge (squash)
 
+## Identifying Library Version Conflicts
+Updates to library versions can create conflicts when another library is dependent on an older version of the given library.  
+1. Run the command `./gradlew dependencies | grep <library name>`.
```

Comment:
Maybe combine the first two steps to say, "After running the command `./gradlew dependencies | grep <library name>`, look for different versions of the library..." That should clear up the redundancy, but allow for a complete step-by-step, here as well.